web/src/components/tasks/business-formation/review/section/ReviewMainSection.tsx

```diff
@@ -1,49 +1,39 @@
 import { UnStyledButton } from "@/components/njwds-extended/UnStyledButton";
 import { LookupStepIndexByName } from "@/components/tasks/business-formation/BusinessFormationStepsConfiguration";
 import { BusinessFormationContext } from "@/contexts/businessFormationContext";
+import { useConfig } from "@/lib/data-hooks/useConfig";
 import { FormationStepNames } from "@/lib/types/types";
 import { scrollToTop } from "@/lib/utils/helpers";
-import { PropsWithChildren, useContext } from "react";
+import { ReactElement, ReactNode, useContext } from "react";
 
 interface Props {
-  buttonText: string;
-  handleButtonClick?: () => void;
   header: string;
   stepName: FormationStepNames;
-  testId: string;
+  testId?: string;
+  children?: ReactNode;
 }
 
-export const ReviewSection: React.FC<PropsWithChildren<Props>> = ({
-  children,
-  buttonText,
-  handleButtonClick,
-  header,
-  stepName,
-  testId,
-}) => {
+export const ReviewMainSection = (props: Props): ReactElement => {
```

Comment:
More of a general nit, but I prefer `ReviewSection` as a name for this component. `ReviewMainSection` to me feels like a generalized wrapper for the whole Review step. Just curious, what was the thought behind the name change?

Reply:
There was logic in place for a `sectionType` property that could be MAIN or SUBSECTION to differentiate the sections by button text or margin properties. In doing so, we created `ReviewSubsection` and renamed `ReviewSection`.

Reply:
Gotcha, I'd advocate for reverting back to `ReviewSection`. I think that name is still more representative.

Reply:
@jphechter My concern about just `ReviewSection` is that the name on its own doesn't communicate that it is part of a "pair" and that it goes hand-in-hand with `ReviewSubSection`.

I'm not tied to "Main" in particular in the name, but I'd argue for something that indicates that this is the "top-level" part of a review section, as contrasted with the subsection level

Reply:
Totally fair. Not a hill I need to die on, I just don't think "main" sounds like something that will be used multiple times side-by-side. At the end of the day, this does not affect how the code renders.